stock_picking_auto_create_lot/static/description/index.html

```diff
@@ -389,7 +389,7 @@ <h1><a class="toc-backref" href="#id1">Configuration</a></h1>
 <p>To configure this module, you need to:</p>
 <ol class="arabic simple">
 <li>Go to a <em>Inventory &gt; Settings &gt; Operation Types</em>.</li>
-<li>Set ???auto create lot??? option for this operation type.</li>
+<li>Set ‘auto create lot’ option for this operation type.</li>
```

Comment:
Changing this is futile, as it's overwritten again by the bot, as it's auto-generated from pieces in `readme` folder. You can even pre-generate it using https://github.com/OCA/maintainer-tools/#readme-generator

Reply:
I try readme generator, thanks

Reply:
I have regenerated the readme with the generator but I have two doubts:

- Should the changelog be changed in the first version of a migration? Only the changes of the 12.0 changes or the previous changes (11.0) are kept too?
- I see that there is a badge 5 with a runbot in the index.html, but I do not see where it comes from. Does that part have to be edited by hand or does the bot modify it?

Reply:
That badge is autogenerated by the README script (run manually with proper arguments or by the bot when this is merged), and it links to the proper branch and repo.

About the changelog, can be considered since the beginning, but personally I'm against that changelog because sometimes there's going to be things that are fixed simultaneously in several branches with a lot of mess in that history. I never put it in my modules (it's optional).